app/boards/shields/reviung39/reviung39.conf

```diff
@@ -0,0 +1,6 @@
+# Uncomment the following lines to enable RGB underglow
+# CONFIG_ZMK_RGB_UNDERGLOW=y
+# CONFIG_WS2812_STRIP=y
+
+# Uncomment the following line to enable deep sleep
+# CONFIG_ZMK_SLEEP=y
```

Comment:
This should probably be removed as it's not a board specific option. We should be setting sane defaults for `ZMK_SLEEP` as well.

Reply:
Better late than never!  Done!